dev/reference/transactions/transaction-pessimistic.md

```diff
@@ -0,0 +1,100 @@
+---
+title: TiDB Pessimistic Transaction Mode
+summary: Learn about TiDB pessimistic transaction mode.
+category: reference
+---
+
+# TiDB Pessimistic Transaction Mode
+
+By default, TiDB implements an optimistic transaction mode, where the commit might fail for a transaction because of transaction conflicts. To improve the commit success rate, you need to modify the application with an automatic retry logic. With the pessimistic transaction mode, you can avoid this potential issue without the need of adding any retry logic to your application.
+
+> **Warning:**
+>
+> Up to now, the pessimistic transaction mode in TiDB is still an **experimental** feature. It is *not recommended to apply it in the production environment*.
```

Comment:
```suggestion
> Pessimistic transactions are an **experimental** feature. We do not recommend use *in production environments*.
```

Reply:
In a technical document, we should try to avoid using "we" unless we are referring to the whole company, that is, PingCAP.

Reply:
You are right. s/We do not/It is not/